### DISCUSSION TITLE
Is there a remark / rehype plugin to convert github links into code blocks?

### DISCUSSION THREAD

**Discussion by faisalsayed10** [remarkjs/remark#1159] [Q&A] [answered]
Basically, if you see some issues, there are links like `https://github.com/faisalsayed10/lmgtfy-slack/blob/f30a5fdb566239f27a076a71d968ac384a5bdd1a/index.js#L13-L17` that will be converted to inline code blocks in markdown. I want to achieve a similar behavior while formatting the markdown. Is there a plugin to do that?
<img width="851" alt="image" src="https://github.com/remarkjs/remark/assets/36627266/623b68c1-3adc-4468-838e-21c5a390318d">

**Comment by wooorm** [comment, marked as answer]
Not yet, but it’s a todo for rehype-github, see the last bullet here: https://github.com/rehypejs/rehype-github/tree/main#packages

**Reply by faisalsayed10** [reply]
Oh, awesome! Do we have an ETA for that by any chance?

**Reply by wooorm** [reply]
No, I'm working on other things now
